src/chainstate/stacks/db/accounts.rs

```diff
@@ -183,6 +183,7 @@ impl StacksChainState {
             let cur_balance = db.get_account_stx_balance(principal);
             let final_balance = cur_balance.checked_add(amount as u128).expect("FATAL: account balance overflow");
             db.set_account_stx_balance(principal, final_balance as u128);
+            info!("{} credited: {} µSTX", principal, final_balance);
```

Comment:
Can you remove the `mu` and replace it with `u`?  Unicode support can be patchy on Linux systems, particularly Docker.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1319/commits/6ebe4e34de3c86261a936e9c62260bef55476cc7